ddl/db_partition_test.go

```diff
@@ -563,6 +563,14 @@ create table log_message_1 (
 				"partition p1 values less than ('A'));",
 			dbterror.ErrRangeNotIncreasing,
 		},
+		{
+			"create table t(d datetime)" +
+				"partition by range columns (d) (" +
+				"partition p0 values less than ('2022-01-01')," +
+				"partition p1 values less than (MAXVALUE), " +
+			        "partition p2 values less than (MAXVALUE));",
```

Comment:
Please format this line, I think you can just do `make dev` and it will reformat it for you.

Reply:
@mjonss reformatted with make dev